fleet/templatetags/feature_icons.py

```
@@ -50,17 +50,19 @@ def feature_icon(feature_name):
+    use_icons = show_icons is not False and str(show_icons).lower() not in ('false', '0', '')
     items = []
     for f in features:
         svg = FEATURE_ICONS.get(f, "")
-        if svg:
+        if svg and use_icons:
             items.append(f'<span class="feature-icon" title="{f}">{svg}</span>')
         else:
             items.append(f)
-    return mark_safe("".join(items))
+    separator = ", " if not use_icons else ""
+    return mark_safe(separator.join(items))
```

Ruff (0.15.21)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)